oak/proto/BUILD

```diff
@@ -143,3 +143,8 @@ proto_library(
     srcs = ["handle.proto"],
     deps = ["@com_google_protobuf//:descriptor_proto"],
 )
+
+proto_library(
```

Comment:
nit: this is probably not needed, seeing that it will only be used by the loader and runtime which uses Prost. There is probably no need to make it available via Bazel to other languages.